### PR TITLE
Add main script to run `integrative-analysis` module

### LINKED ISSUE #75: To add main script to run `integrative-analysis` module
## What are the scientific goals of the analysis?
To add main script to run `integrative-analysis` module

### PR DESCRIPTION
**Pull request by AntoniaChroni**
# Purpose/implementation Section

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update.

## What scientific question/module of analysis is your analysis addressing?

To add the main script to run the `integrative-analysis` module.

## What GitHub issue does your pull request address?

Closes #75.

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?
 
Please review the code and documentation for clarity.

## Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/integrative-analysis/01-integrative-analysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,366 @@
+---
+title: "Integrative analysis for sc-/sn-RNA-Seq Analysis in 10X Genomics data"
+author: "Antonia Chroni for SJCRH DNB_BINF_Core"
+author: "Antonia Chroni for SJCRH DNB_BINF_Core"
```
Comment (sharonfreshour) [inline analyses/integrative-analysis/01-integrative-analysis.Rmd:4]: author line is duplicated

File: analyses/integrative-analysis/01-integrative-analysis.Rmd [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+(2) [Harmony](https://portals.broadinstitute.org/harmony/) integration workflow by [Korsunsky et al., 2019](https://www.nature.com/articles/s41592-019-0619-0), and
+(3) [Liger](https://welch-lab.github.io/liger/articles/liger_with_seurat.html) integration workflow by [Welch et al., 2019](https://doi.org/10.1016/j.cell.2019.05.006).
+
+Different number of dimensions for different integration algorithms (Seurat, Harmony, Liger, etc.) is common and depending on the algorithm they come with their corresponding variables depending on how the algorithm is being used. Each of these integration algorithm takes into account typically the same variables but the math they use to transform them is so different that you can't use the same number of resolutions. Even though they use the same word, resolution doesn't actually effect each algorithm the same way and their maximums can be different. 30 for Seurat, 30 for Harmony and 20 for Liger give the most similar integration results when all other variables are taken into account while also not wasting too much compute time.
```
Comment (sharonfreshour) [inline analyses/integrative-analysis/01-integrative-analysis.Rmd:151]: The first sentence is a bit hard to follow. I tried to make a suggestion to clarify it, but I'm not sure what will sound best.

Also, are dimension and resolution being used interchangeably here? If so, perhaps it is better to stick with using dimension throughout for clarity. And to not cause confusion with the resolution parameter which is used for clustering.

```suggestion
Using a different number of dimensions for different integration algorithms (Seurat, Harmony, Liger, etc.) is common and their corresponding variables depend on the algorithm and how the algorithm is being used. Each of these integration algorithms takes into account typically the same variables but the math they use to transform them is so different that you can't use the same number of resolutions. Even though they use the same word, resolution doesn't actually effect each algorithm the same way and their maximums can be different. 30 for Seurat, 30 for Harmony and 20 for Liger give the most similar integration results when all other variables are taken into account while also not wasting too much compute time.
```

**Review by sharonfreshour — APPROVED** [review]
I suggested a change to clarify some explanations of the script and to remove one duplicated line, but otherwise the code itself looks logical.

File: analyses/integrative-analysis/01-integrative-analysis.Rmd (hunk 1 above)
Comment (sharonfreshour) [reply analyses/integrative-analysis/01-integrative-analysis.Rmd:4]:
```suggestion
```

Comment (AntoniaChroni) [reply analyses/integrative-analysis/01-integrative-analysis.Rmd]: Resolved here b5c1ce8017ebfec75b0df9dc348a7a2922970556

**Review by sharonfreshour — APPROVED** [review]
Updates look good, the documentation is clear now.
